clients/lightrider/lightrider-entry.js

```diff
@@ -25,9 +25,9 @@ const LR_PRESETS = {
  * If configOverride is provided, lrDevice and categoryIDs are ignored.
  * @param {Connection} connection
  * @param {string} url
- * @param {LH.Flags} flags Lighthouse flags, including `output`
+ * @param {LH.Flags} flags Lighthouse flags
  * @param {{lrDevice?: 'desktop'|'mobile', categoryIDs?: Array<string>, logAssets: boolean, configOverride?: LH.Config.Json}} lrOpts Options coming from Lightrider
- * @return {Promise<string|Array<string>|void>}
+ * @return {Promise<string|void>}
```

Comment:
is `void` still a return possibility? (I think it can be dropped now)

Reply:
> is `void` still a return possibility?

Oh, yes it is for the `if (!results) return` case. Should it throw to get a runtime error when that happens? Weird to return nothing if Lighthouse was acting poorly (I think it's actually a `// should never happen` case to make tsc happy).